app/elements/calendar-app.html

```diff
@@ -176,6 +180,13 @@
       renderColor: function(event, element) {
         var edition = (event.assignment ? event.assignment : event).course.edition;
 
+        var node = document.createElement("span");
+        var textnode = document.createTextNode(" [" + edition.courseId + "] ");
+        node.appendChild(textnode);
+        node.style.float = "left";
+        node.style.padding = "0 5px 0 0";
+        element[0].firstChild.children[1].appendChild(node);
```

Comment:
All right I probably figured out a solution. In [`eventDataTransform`](https://github.com/Mentoraat/plannings-tool-client/blob/664b81be21b09121cf19e61a95659edb1ae13733/app/elements/calendar-app.html#L90) you can set the [`event.title`](http://fullcalendar.io/docs/event_data/Event_Object/) to `event.title + '[' + (event.assignment ? event.assignment : event).course.edition.courseId + ']'`